cypress/testcases/rancher/rancher_integration.spec.ts

```diff
@@ -138,12 +139,30 @@ describe('Harvester import Rancher', function () {
     it('Check Harvester Cluster Status', { baseUrl: constants.rancherUrl }, () => {
         // cy.login();
         cy.visit('/');
-        cy.wait(5000);
+        cy.wait(constants.timeout.timeout);
+
+        rancher.rancherLogin();
+
+        rancher.visit_virtualizationManagement();
+
+        rancher.checkState(rData.harvester_cluster_name);
```

Comment:
It starts a timer loop that checks for when the status changes to active. It will timeout eventually. I also added in the checkState for this one too since it's never a bad idea to be sure.